bench_runner/result.py

```diff
@@ -779,6 +780,23 @@ def track(it, *_args, **_kwargs):
                 ),
             )
 
+            compare_to_default = (
+                config.get_bench_runner_config()
+                .get("bases", {})
+                .get("compare_to_default", [])
+            )
+
+            if "NOGIL" in compare_to_default and result.flags == ["NOGIL"]:
```

Comment:
Can we make this generic, so it would work with any flag?

```suggestion
            if any(result.flags == [flag] for flag in compare_to_default):
```

Reply:
If compare_to_default = ["NOGIL", "JIT"], then we need two different `base` name for `find_match`.